src/dreamer/dream_scheduler.py

```
@@ -160,7 +160,7 @@ async def _delayed_dream(
                 )
 
         except asyncio.CancelledError:
-            logger.info(f"Dream task cancelled for {work_unit_key}")
+            logger.debug(f"Dream task cancelled for {work_unit_key}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)